docs/content/en/blog/heir_meeting_2024-11-26.md

```diff
@@ -0,0 +1,14 @@
+---
+title: HEIR meeting 2024-11-26
+date: 2024-11-26T00:00:00+00:00
+linkTitle: HEIR meeting 2024-11-26
+description: >
+  Notes from the 2024-11-26 HEIR meeting.
+author: '[Jeremy Kun](https://jeremykun.com)'
+---
+
+Here are the
+[notes](https://docs.google.com/document/d/10WTpdVaLz4mgmTwlJ3lVMGD4qmiNEl2Uvt3R3h0XFmc/edit?hl=en&forcehl=1&tab=t.0#heading=h.w3lar3vbuip1),
+[video recording](https://drive.google.com/file/d/1emDpb1B5ZEMlAgJXSjAe25vz5RgPfBm7/view?usp=sharing),
```

Comment:
the link doesn't seem to be public (yet)?

Reply:
Oh - I was always downloading and uploading to my personal drive. Let me do that and update the link.

(My personal corp didn't allow me to record meetings)

Reply:
@j2kun here's the public link: https://drive.google.com/file/d/13yvX_ap09tTqwJsdABDf3WDl79GF3bJx/view?usp=sharing